mlir/lib/Dialect/Rock/Transforms/GridwiseGemmToBlockwise.cpp

```diff
@@ -428,21 +229,23 @@ static FailureOr<Value> wrapLDSBufferForStore(OpBuilder &b, Location loc,
 static ArrayAttr globalVectorLayout(OpBuilder &b, Location loc, StringRef dName,
                                     int64_t kPerThread, int64_t dPerThread,
                                     int64_t kpack, GemmDimension vectorDim) {
-  int64_t kOuter = kPerThread / std::min(kPerThread, kpack);
+  int64_t kpackPerThread = std::min(kPerThread, kpack);
+  int64_t kOuter = kPerThread / kpackPerThread;
+
   int64_t dataPerThread = kPerThread * dPerThread;
 
   TopDownTMBuilder splitIter(b, {"iter"}, {dataPerThread});
   if (vectorDim == GemmDimension::K)
-    splitIter.merge({dName, "k", "kpack"}, {0, 1, 2}, "iter",
-                    {dPerThread, kOuter, kpack});
+    splitIter.merge({dName, "k", "kpack_thread"}, {0, 1, 2}, "iter",
```

Comment:
Nit: the rest of the names in here don't include the thread-ness, we could probably just call this "kpack"?

Reply:
The reason why I added `kpack_thread` is to just give a reminder to who read the transform map that this dimension isn't kpack (and we don't guarantee the size of it is equal to kapck). I was having this assumption when I was doing such so having a hard time to realize why we need `kpackPerThread` abstraction. 

I think I get it from your side that the transformations already have tid in them so it is implied, but I'm just trying to make it a different term for differentiation.

Reply:
Not a bad point, and I don't see any reason not to do it your way